<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StakeForCharityDialog</class>
 <widget class="QDialog" name="StakeForCharityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stake For Charity</string>
  </property>
  <widget class="QLineEdit" name="charityPercentEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter whole numbers 1 - 50</string>
   </property>
   <property name="placeholderText">
    <string>Enter % to Give (1-50)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="charityAddressEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Address to Send to</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>591</width>
     <height>81</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Stake For Charity allows you to automatically send up to 50% of your stake to another HoboNickels address after it matures.

Please enter percentage to give and HoboNickels address to give to. Min and Max amount are optional.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="charityMinEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter whole numbers</string>
   </property>
   <property name="placeholderText">
    <string>Enter Min Amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="charityMaxEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter whole numbers</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Max Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="enableButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Check and turn on Stake For Charity</string>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disableButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Turn off Stake For Charity</string>
   </property>
   <property name="text">
    <string>Disable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addressBookButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>28</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose an address from the address book</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../bitcoin.qrc">
     <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
